tests/snowpark/test_jobs.py

```diff
@@ -10,32 +10,29 @@ def test_create_job(mock_connector, runner, mock_ctx):
     ctx = mock_ctx()
     mock_connector.return_value = ctx
 
-    with NamedTemporaryFile(
-        prefix="spec", suffix="yaml", dir=Path(__file__).parent
-    ) as fh:
-        name = fh.name
-        result = runner.invoke(
+    with TemporaryDirectory() as temp_dir:
+        filepath = os.path.join(temp_dir, "test")
+        with open(filepath, "w") as fh:
+            fh.write("test data")
```

Comment:
Is this a valid data we expect? Can we make it more like expected spec file?

Reply:
done